### ISSUE TITLE
[BUG] "TZ" variable only changes the name of the timezone and not the time.

### ISSUE THREAD

**Issue by JohanVlugt** [mbround18/valheim-docker#736]
Local time in my location: `09:58`
command output: `date`
```cmd
Sat Sep 30 9:58:41 CEST 2023
```
Docker compose:
```yaml
    environment:
      TZ: "Europe/Amsterdam"
```
For example when I set the time for a location in the EU it will show this message.

```cmd
Valheim Server - Sat Sep 30 07:58:27 PM Europe 2023
```
When changed it to the default in the US.
```cmd
Valheim Server - Sat Sep 30 07:59:51 PM America 2023
```

What I want to happen: is that the time/backups are actually synced to my own time zone.

What actually happens: is that it seems like it's set to UTC.

Thanks for this project!

**Comment by mbround18** [comment]
who needs time zones anway ;) jk jk ill take a look! maybe its missing a command

**Comment by FooDeas** [comment]
I just ran into the problem some minutes ago when I was wondering why the update and backup jobs were not running at the expected times. It follows a small pull request in a few minutes.

**Comment by mbround18** [comment]
@FooDeas can you verify this is now working on v2.3.0?

**Comment by FooDeas** [comment]
Same here. 2.3.0 is not working.

**Comment by FooDeas** [comment]
The issue may be the following:
```
valheim_1  | [ODIN][DEBUG] - Base Command: Command {
valheim_1  | [ODIN][DEBUG] -     program: "/home/steam/valheim/valheim_server.x86_64",
valheim_1  | [ODIN][DEBUG] -     args: [
valheim_1  | [ODIN][DEBUG] -         "/home/steam/valheim/valheim_server.x86_64",
valheim_1  | [ODIN][DEBUG] -         "-name <...>",
valheim_1  | [ODIN][DEBUG] -         "-port 2456",
valheim_1  | [ODIN][DEBUG] -         "-world <...>",
valheim_1  | [ODIN][DEBUG] -         "-public 0",
valheim_1  | [ODIN][DEBUG] -         "-nographics -batchmode",
valheim_1  | [ODIN][DEBUG] -         "-password <...>",
valheim_1  | [ODIN][DEBUG] -         "-savedir /home/steam/.config/unity3d/IronGate/Valheim",
valheim_1  | [ODIN][DEBUG] -     ],
valheim_1  | [ODIN][DEBUG] -     env: CommandEnv {
valheim_1  | [ODIN][DEBUG] -         clear: false,
valheim_1  | [ODIN][DEBUG] -         vars: {
valheim_1  | [ODIN][DEBUG] -             "SteamAppId": Some(
valheim_1  | [ODIN][DEBUG] -                 "892970",
valheim_1  | [ODIN][DEBUG] -             ),
valheim_1  | [ODIN][DEBUG] -         },
valheim_1  | [ODIN][DEBUG] -     },
valheim_1  | [ODIN][DEBUG] -     cwd: Some(
valheim_1  | [ODIN][DEBUG] -         "/home/steam/valheim",
valheim_1  | [ODIN][DEBUG] -     ),
valheim_1  | [ODIN][DEBUG] -     create_pidfd: false,
valheim_1  | [ODIN][DEBUG] - }
```
As you can see, you have the **executable additionally set as argument**. That breaks the command.
My Verification is the command executed in the container environment:
`LD_LIBRARY_PATH=/home/steam/valheim/linux64 SteamAppId=892970 /home/steam/valheim/valheim_server.x86_64 -name <...> -port 2456 -world <...> -public 0 -nographics -batchmode -password <...> -savedir /home/steam/.config/unity3d/IronGate/Valheim`
That still works as expected.

**Comment by FooDeas** [comment]
When I remove `TYPE=BepInEx` and the `MOD` argument. The server starts as expected!

(previous post deleted)

**Comment by FooDeas** [comment]
I also wanted to have a look - but I don't see why it doesn't work.
```
valheim_1  | [ODIN][DEBUG] - Base Command: Command {
valheim_1  | [ODIN][DEBUG] -     program: "/home/steam/valheim/valheim_server.x86_64",
valheim_1  | [ODIN][DEBUG] -     args: [
valheim_1  | [ODIN][DEBUG] -         "/home/steam/valheim/valheim_server.x86_64",
valheim_1  | [ODIN][DEBUG] -         "-name Servername",
valheim_1  | [ODIN][DEBUG] -         "-port 2456",
valheim_1  | [ODIN][DEBUG] -         "-world Worldname",
valheim_1  | [ODIN][DEBUG] -         "-public 0",
valheim_1  | [ODIN][DEBUG] -         "-nographics -batchmode",
valheim_1  | [ODIN][DEBUG] -         "-password Worldpassword",
valheim_1  | [ODIN][DEBUG] -         "-savedir /home/steam/.config/unity3d/IronGate/Valheim",
valheim_1  | [ODIN][DEBUG] -     ],
valheim_1  | [ODIN][DEBUG] -     env: CommandEnv {
valheim_1  | [ODIN][DEBUG] -         clear: false,
valheim_1  | [ODIN][DEBUG] -         vars: {
valheim_1  | [ODIN][DEBUG] -             "SteamAppId": Some(
valheim_1  | [ODIN][DEBUG] -                 "892970",
valheim_1  | [ODIN][DEBUG] -             ),
valheim_1  | [ODIN][DEBUG] -         },
valheim_1  | [ODIN][DEBUG] -     },
valheim_1  | [ODIN][DEBUG] -     cwd: Some(
valheim_1  | [ODIN][DEBUG] -         "/home/steam/valheim",
valheim_1  | [ODIN][DEBUG] -     ),
valheim_1  | [ODIN][DEBUG] -     create_pidfd: false,
valheim_1  | [ODIN][DEBUG] - }
valheim_1  | [ODIN][DEBUG] - Executable: /home/steam/valheim/valheim_server.x86_64
valheim_1  | [ODIN][INFO]  - Launching Command...
...
valheim_1  | 10/12/2023 20:29:08: Error bad password:The password is too short.
...
```
I don't see, what's the issue with the launcher...

**Comment by mbround18** [comment]
That should work, it looks like an issue with the command launching module, i have a fix for this at #747

**Comment by mbround18** [comment]
tested and working in [v2.3.1](https://github.com/mbround18/valheim-docker/releases/tag/v2.3.1)
